recipes/opengl/all/conanfile.py

```diff
@@ -63,6 +63,10 @@ def system_requirements(self):
                 package_tool.install(update=True, packages=p)
 
     def package_info(self):
+        # Workaround for #2311 until a better solution can be found
```

Comment:
```suggestion
        # TODO: Workaround for #2311 until a better solution can be found
```

nit: Add a keyword so that editors make such things more easily discoverable